dashboards/circle.erb

```diff
@@ -113,6 +119,9 @@
     <li data-row="6" data-col="6" data-sizex="1" data-sizey="1">
       <div data-id="circle-ci-ministryofjustice-manage-intelligence-api-main" data-view="CircleCi" data-title="Manage Intelligence API"></div>
     </li>
+    <li data-row="6" data-col="7" data-sizex="1" data-sizey="1">
+      <div data-id="circle-ci-ministryofjustice-hmpps-restricted-patients-api-main" data-view="CircleCi" data-title="HMPPS Restricted Patients API"></div>
```

Comment:
this was missing `-main` so not working. 

Shunted this inline with the rest of the tiles too.